packages/web3-shared/base/src/specs/index.ts

```diff
@@ -609,18 +607,42 @@ export interface DomainResult<ChainId> extends Result<ChainId> {
 }
 
 export interface FungibleTokenResult<ChainId, SchemaType> extends Result<ChainId> {
-    address: string
+    //  This id on the provider platform
+    id?: string
```

Comment:
Why UI needs this?

Reply:
The DSearch API return the tidy data. The caller need to get detail from data provider by id.